### PR TITLE
Feature/dev initial play

### PR DESCRIPTION
**Pull request by karankarpe92**

### CODE REVIEW

File: .flutter-plugins [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+firebase_core_web=C:\\Users\\KAKARP\\Documents\\Projects\\Android\\Flutter SDK\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\firebase_core_web-0.1.1+2\\
+google_sign_in=C:\\Users\\KAKARP\\Documents\\Projects\\Android\\Flutter SDK\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\google_sign_in-4.5.5\\
+google_sign_in_web=C:\\Users\\KAKARP\\Documents\\Projects\\Android\\Flutter SDK\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\google_sign_in_web-0.9.2\\
+path_provider=C:\\Users\\KAKARP\\Documents\\Projects\\Android\\Flutter SDK\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\path_provider-1.6.22\\
+path_provider_linux=C:\\Users\\KAKARP\\Documents\\Projects\\Android\\Flutter SDK\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\path_provider_linux-0.0.1+2\\
+path_provider_macos=C:\\Users\\KAKARP\\Documents\\Projects\\Android\\Flutter SDK\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\path_provider_macos-0.0.4+4\\
+path_provider_windows=C:\\Users\\KAKARP\\Documents\\Projects\\Android\\Flutter SDK\\flutter\\.pub-cache\\hosted\\pub.dartlang.org\\path_provider_windows-0.0.4+1\\
```
Comment (harshLFC) [inline .flutter-plugins:12]: Should we git ignore this file?

File: .flutter-plugins-dependencies [hunk 2]
```diff
@@ -1 +1 @@
...
..."dependencies":[]},{"name":"firebase_core_web","path":"C:\\\\Users\\\\KAKARP\\\\Documents\\\\Projects\\\\Android\\\\Flutter SDK\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\firebase_core_web-0.1.1+2\\\\","dependencies":[]},{"name":"google_sign_in_web","path":"C:\\\\Users\\\\KAKARP\\\\Documents\\\\Projects\\\\Android\\\\Flutter SDK\\\\flutter\\\\.pub-cache\\\\hosted\\\\pub.dartlang.org\\\\google_sign_in_web-0.9.2\\\\","dependencies":[]}]},"dependencyGraph":[{"name":"firebase_analytics","dependencies":["firebase_analytics_web"]},{"name":"firebase_analytics_web","dependencies":[]},{"name":"firebase_auth","dependencies":["firebase_core"]},{"name":"firebase_core","dependencies":["firebase_core_web"]},{"name":"firebase_core_web","dependencies":[]},{"name":"google_sign_in","dependencies":["google_sign_in_web"]},{"name":"google_sign_in_web","dependencies":[]},{"name":"path_provider","dependencies":["path_provider_macos","path_provider_linux","path_provider_windows"]},{"name":"path_provider_linux","dependencies":[]},{"name":"path_provider_macos","dependencies":[]},{"name":"path_provider_windows","dependencies":[]}],"date_created":"2020-11-03 10:54:17.154732","version":"1.20.2"}
```
Comment (harshLFC) [inline .flutter-plugins-dependencies:1]: Same with this ?

File: .flutter-plugins (hunk 1 above)
Comment (karankarpe92) [reply .flutter-plugins:12]: Yes

File: .flutter-plugins-dependencies (hunk 2 above)
Comment (karankarpe92) [reply .flutter-plugins-dependencies:1]: Yes

File: .flutter-plugins (hunk 1 above)
Comment (karankarpe92) [reply .flutter-plugins:12]: Yes can ignore

File: .flutter-plugins-dependencies (hunk 2 above)
Comment (karankarpe92) [reply .flutter-plugins-dependencies:1]: Yes can ignore

File: lib/screens/fixtures/fixture-list.dart [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+  factory Match.fromJson(Map<String, dynamic> json) {
+    return Match(
+      id: json['_id'],
+      away_team_name: json['away_team_name'],
+      away_team_score:
+          json['away_team_score'] == null ? 0 : json['away_team_score'],
+      date: json['date'],
+      home_team_name: json['home_team_name'],
+      home_team_score:
+          json['home_team_score'] == null ? 0 : json['home_team_score'],
```
[the remark this answers is not in the document]
Comment (karankarpe92) [reply lib/screens/fixtures/fixture-list.dart:61]: For score data type is int and cannot accept null

File: lib/screens/fixtures/fixture-list.dart [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: <Widget>[
+        Expanded(
+          flex: 4,
+          child: TeamDetailWidget(
+            teamName: homeTeamName,
+            imageName: "icon-soccer-ball.png",
```
[the remark this answers is not in the document]
Comment (karankarpe92) [reply lib/screens/fixtures/fixture-list.dart:184]: Given default image as don't have all images

File: lib/screens/fixtures/fixture-list.dart [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+        ),
+        Expanded(
+          flex: 1,
+          child: Text(""),
+        ),
+        Expanded(
+          flex: 4,
+          child: TeamDetailWidget(
+            teamName: awayTeamName,
+            imageName: "icon-soccer-ball.png",
```
[the remark this answers is not in the document]
Comment (karankarpe92) [reply lib/screens/fixtures/fixture-list.dart:195]: Given default image as don't have all images

File: lib/screens/fixtures/fixture-list.dart [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+          : List<Match>.from(
+              json["matches"].map(
+                (x) => Match.fromJson(x),
+              ),
+            ),
+    );
+  }
+}
+
+class Match {
```
[the remark this answers is not in the document]
Comment (karankarpe92) [reply lib/screens/fixtures/fixture-list.dart:32]: Already maintained a seperate structure

**Pull request merged**